modules/policy-criteria.adoc

```diff
@@ -35,518 +35,820 @@ To use logical operators `AND`, `OR`, and `NOT` for creating security policies,
 However, on earlier versions you can still use regular expressions for the fields listed in the *Regular expressions* column.
 ====
 
-[cols="<,<,<,^,^,^,<"]
+[cols="<,<,<,<,^,^,^,<"]
 |===
-| *Attribute* | *Description* | *RHACS version* | *Regular expressions* | *NOT* | *AND, OR* | *Phase*
+| *Attribute* | *Description* | *JSON Attribute* | *Allowed Values* | *Regular expressions* | *NOT* | *AND, OR* | *Phase*
 
-| Namespace
-| The name of the namespace.
-| 3.0.51 and newer
-| ✓
-| ✓
-| ✓
-| Deploy
+8+| *Section: Image registry*
 
 | Image Registry
 | The name of the image registry.
-| All
+| Image Registry
+| String
 | ✓
 | ✓
 | ✓
 | Deploy
 
-| Image Remote
+| Image Name
 | The full name of the image in registry, for example `library/nginx`.
-| All
+| Image Remote
+| String
 | ✓
 | ✓
 | ✓
 | Deploy
 
 | Image Tag
 | Identifier for an image.
-| All
+| Image Tag
+| String
 | ✓
 | ✓
 | ✓
 | Deploy
 
-| Days since image was created
-| The number of days from image creation date.
-| All
+| Image Signature
+| The list of signature integrations you can use to verify an image's signature. Create alerts on images that either do not have a signature or their signature is not verifiable by at least one of the provided signature integrations.
+| Image Signature Verified By
+| A valid ID of an already configured image signature integration
+| ✕
+| ✕
+| ! `OR` only
+| Deploy
+
+8+| *Section: Image contents*
+
+| Image age
+| The minimum number of days from image creation date.
+| Image Age
+| Integer (whole numbers only)
 | ✕
 | ✕
 | ✕
 | Build
 
-| Days since image was last scanned
-| The number of days since the last image scan.
-| All
+| Image scan age
+| The minimum number of days since the image was last scanned.
+| Image Scan Age
+| Integer (whole numbers only)
 | ✕
 | ✕
 | ✕
 | Build
 
+| Image User
+| Matches the USER directive in the Dockerfile. See [ Docker documentation Hyperlink] for details
```

Comment:
Can we see if this would work? (should automatically create hyperlink)
```suggestion
| Matches the USER directive in the Dockerfile. See https://docs.docker.com/engine/reference/builder/#user for details.
```